.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
```

Comment:
Great job @wassimchakib 🤝

- [ ] You have done well adding the `linters.yml` file. However, your project still misses other linter files. The `.hintrc` file and the `.stylelintrc` file. You can find these files in the [Linter files](https://github.com/microverseinc/linters-config/tree/master/html-css).